network.go

```diff
@@ -250,7 +255,7 @@ func (net *Network) ConnectHemispheres() {
                 excitatory = true
             }
             toNodes[node] = &ConnInfo{
-                Strength: RandFloat(0.75, 1.75),
+                Strength: RandFloat(0.75, 1.75), // magic numbers
```

Comment:
`MAGIC_CONSTANT_1`
`MAGIC_CONSTANT_2`
...
`MAGIC_CONSTANT_i`

perfect.

In all seriousness, I should get around to doing this but I've been generating a lot of really cool data (and graphing it) so this isn't at the top of my mind - maybe make an issue for this?  (since I do actually go through those)